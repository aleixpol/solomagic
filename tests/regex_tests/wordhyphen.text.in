\ref something0
\ELANParticipant unknown

\mt somethingsomething nonovul- -nonovulu somethingsomething
\ma 
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:13.510
\ELANEnd 00:00:15.512



\ref something1
\ELANParticipant someone

\mt somethingsomething XXXnonovulu- nono-vulu somethingsomething
\ma 
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:15.512
\ELANEnd 00:00:18.515

