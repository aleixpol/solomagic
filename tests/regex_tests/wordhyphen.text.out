\ref something0
\ELANParticipant unknown

\mt somethingsomething nonovul- -nonovulu somethingsomething
\ma somethingsomething * -nonovulu somethingsomething
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:13.510
\ELANEnd 00:00:15.512



\ref something1
\ELANParticipant someone

\mt somethingsomething XXXnonovulu- nono-vulu somethingsomething
\ma somethingsomething * nono-vulu somethingsomething
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:15.512
\ELANEnd 00:00:18.515

